app/presenters/publishing_api_presenters/html_attachment.rb

```diff
@@ -0,0 +1,68 @@
+require_relative "../publishing_api_presenters"
+
+class PublishingApiPresenters::HtmlAttachment < PublishingApiPresenters::Item
+  def links
+    {
+      parent: [
```

Comment:
Had a quick chat with @gpeng. To summarise: use of `parent` here is the correct thing. 

As we see it now, the parent will on most cases be used for the breadcrumb. In the case of this publication, the attachable seems to be the logical/only candidate to use as the breadcrumb parent.

So :+1: on this.